stubs/pyinstaller/PyInstaller/depend/analysis.pyi

```diff
@@ -2,17 +2,25 @@
 
 # The documentation explicitely mentions that "Normally you do not need to know about the module-graph."
 # However, some PyiModuleGraph typed class attributes are still documented as existing in imphookapi.
-from _typeshed import Incomplete
+from _typeshed import Incomplete, StrOrBytesPath, SupportsKeysAndGetItem
+from collections.abc import Iterable
+from typing_extensions import TypeAlias
+
+from PyInstaller.lib.modulegraph.modulegraph import Alias, Node
+
+_LazyNode: TypeAlias = Iterable[Node] | Iterable[str] | Alias | None
+# from altgraph.Graph import Graph
+_Graph: TypeAlias = Incomplete
 
 class PyiModuleGraph:  # incomplete
     def __init__(
         self,
         pyi_homepath: str,
-        user_hook_dirs=...,
-        excludes=...,
-        path: Incomplete | None = ...,
-        replace_paths=...,
-        implies=...,
-        graph: Incomplete | None = ...,
+        user_hook_dirs: Iterable[StrOrBytesPath] = ...,
```

Comment:
Bytes paths don't work: `PyInstaller.depend.analysis.PyiModuleGraph("x", [b"y"])` throws a TypeError.

Reply:
> TypeError: Can't mix strings and bytes in path components

This is probably the case in other parts of the stubs as well 😓 I'll go back through other `StrOrBytesPath` and make a PR to fix them